manage.py

```diff
@@ -140,6 +140,56 @@ def _add_live_test_subparser(subparsers):
     )
 
 
+def _add_validate_schema_subparser(subparsers):
+    """Add the validate-schemas subparser: manage.py validate-schemas [options]"""
+    schema_validation_usage = 'manage.py validate-schemas [options]'
+    schema_validation_description = ("""
+StreamAlertCLI v{}
+Run end-to-end tests that will attempt to send alerts
+
+Available Options:
+
+    --test-files         Name(s) of test files to validate, separated by spaces (not full path)
```

Comment:
Isn't it usually the standard to separate multiple arguments in CLI by commas instead of spaces? Each space-delimited token should be its own separate argument